crates/api_common/src/utils.rs

```diff
@@ -94,9 +96,10 @@ pub fn is_top_mod(
       .map(|cm| cm.moderator.id)
       .unwrap_or(PersonId(0))
   {
-    Err(LemmyErrorType::NotTopMod)?;
```

Comment:
@Nutomic I found a bunch of these, that clippy fixed earlier, that could be scary, and I have no idea why rust isn't catching them. 

Maybe I'm being paranoid, but it seems like the `;` means that it wouldn't actually return the Err, even if it hit an Error result, and just continue on to the `Ok(())` condition. 

This is why default Ok returns are really scary to me.

Reply:
the ? does an early return so the ; doesn't matter

Reply:
Gotcha, then rust is doing a good job of keeping us safe there.